container/container_contract.go

```diff
@@ -93,6 +95,32 @@ func _deploy(data interface{}, isUpdate bool) {
 	if isUpdate {
 		args := data.([]interface{})
 		common.CheckVersion(args[len(args)-1].(int))
+
```

Comment:
There has to be some check for the current contract version here, if it's `>= some` this migration is not needed.

Reply:
Our current policy is that we remove this code after release and update "min updating from" versions in `common` package. This way we can update contract multiple times, but do not have complex in `_deploy`.

Reply:
Not very user-friendly, upgrading contracts multiple times in no fun especially given that some node versions work fine with some contract versions and don't work with others.